### ISSUE TITLE
Show downstream tasks

### ISSUE THREAD

**Issue by geekflyer** [cirruslabs/cirrus-ci-web#99]
Currently cirrus already displays in the UI all the previous tasks a particular task is depends on.
I think it would be also good if we display the tasks that depend on the current task (i.e. that will be triggered after the currently viewed task is complete).

Currently I find myself always navigating back to "all tasks" in order to get to the next task which is kind unintuitive and time-consuming, especially when one has a a lot of tasks.

In other words: Please show both the upstream and downstream dependencies / tasks of a task.

This optionally (really this could be a separate enhancement) could be displayed as a graph similar to this:

![image](https://user-images.githubusercontent.com/1221897/62504614-33a2a380-b7ae-11e9-9a8b-80ed58a5a193.png)

**Comment by fkorotkov** [comment]
Been thinking about the second part for a while. I think it will also cool to use `scheduledTimestamp` for X coordinate on the graph plot so in case of concurrency limits for OSS projects it will be nicely illustrated.

**Comment by fkorotkov** [comment]
Probably we can add "Dependants" section **after** all the instructions. Kind of making the linear history.

<img width="1207" alt="Screenshot 2022-11-03 at 7 54 40 AM" src="https://user-images.githubusercontent.com/989066/199715321-494bd648-1c1a-49db-bd39-a852c78cdcc1.png">
